Formula/xcb-util-keysyms.rb

```diff
@@ -0,0 +1,29 @@
+class XcbUtilKeysyms < Formula
+  desc "Standard X constants and conversion to/from keycodes"
+  homepage "https://xcb.freedesktop.org"
+  url "https://xcb.freedesktop.org/dist/xcb-util-keysyms-0.4.0.tar.bz2"
+  sha256 "0ef8490ff1dede52b7de533158547f8b454b241aa3e4dcca369507f66f216dd9"
+  license "X11"
+
+  depends_on "pkg-config" => [:build, :test]
+  depends_on "util-macros" => :build
+  depends_on "libxcb"
+
+  def install
+    args = %W[
```

Comment:
I think we can add the args directly after the ./configure, as there is no need to have an intermediate variable.